Integrations/integration-DomainTools.yml

```diff
@@ -0,0 +1,541 @@
+description: Domain name, DNS and Internet OSINT-based cyber threat intelligence and
+  cybercrime forensics products and data
+configuration:
+- display: DomainTools API URL
+  name: server
+  defaultvalue: http://api.domaintools.com/
+  type: 0
+  required: true
+- display: API Username
+  name: username
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API Key
+  name: key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Open authentication (less secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |-
+    var toCamelCase = function(string){
+        var str = ' '+string.trim();
+        str=str.replace(/ ([a-z,A-Z])/g, function (g) { return g[1].toUpperCase(); });
+        return str;
+    };
+
+    var sendRequest = function(url) {
+        var res = http(
+            url,
+            {
+                Method: 'GET',
+                Headers: {
+                    Accept: ['application/json']
+                }
+            }
+        );
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+        return JSON.parse(res.Body);
```

Comment:
try { return JSON.parse(res.Body); }; throw 'Failed to parse JSON';
here, so if the response is not json we will get a clear error